painttiming.bs

```diff
@@ -6,7 +6,10 @@ Level: 1
 ED: https://w3c.github.io/paint-timing/
 TR: https://www.w3.org/TR/paint-timing/
 Status: ED
-Editor: Shubhie Panicker, Google https://google.com, panicker@google.com
+Editor: Nicolás Peña Moreno, Google https://google.com, npm@chromium.org
+        Noam Rosenthal
+        Yoav Weiss, Google https://google.com, yoavweiss@chromium.org
```

Comment:
Looked at my [list of commits](https://github.com/w3c/paint-timing/commits?author=yoavweiss), and I don't think it justifies adding me as an editor. While I did do a bunch of [PR reviews](https://github.com/w3c/paint-timing/search?q=reviewed-by%3Ayoavweiss&type=Issues), I'm happy with you two being the active editors.

Reply:
Ok, removed you